### PR TITLE
 Use pytest directly from recipe instead of astropy.test

### PR DESCRIPTION
**Pull request by MridulS**
This should hopefully work, let's see if the CI correctly picks it up :)

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by MridulS** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -68,6 +68,11 @@ test:
     - samp_hub --help
     - volint --help
     - wcslint --help
+    # We run a subset of the tests which are the most likely to have
+    # issues because they rely on C extensions and bundled libraries
+    - pytest --pyargs astropy.wcs astropy.io.fits
+    # Skipping most tests on aarch64/ppc64le because they take too long
+    - pytest --pyargs astropy.time astropy.convolution astropy.io.ascii # [not (ppc64le or aarch64)]
   imports:
     - astropy
```
Comment (Cadair) [inline recipe/meta.yaml:77]: Can you add the other imports from run_test?

**Comment by Cadair** [comment]
Thanks a lot for doing this. I would like to get all the package split stuff done first before we merge this though.

**Comment by MridulS** [comment]
> Thanks a lot for doing this. I would like to get all the package split stuff done first before we merge this though.

Sure thing! Feel free to close this too if it's easier to just incorporate these changes in another PR :)

**Review by bsipocz — APPROVED** [review]
Looks all good, thanks! 

@Cadair - yours should go in first, but the diff is too complicated to do a review from my phone. Hopefully someone else can give you a review of it while I'm travelling.

**Review by Cadair — APPROVED** [review]

**Comment by Cadair** [comment]
Thanks a lot @MridulS :partying_face:

**Pull request merged**